src/Agent.Worker/Handlers/NodeHandler.cs

```diff
@@ -1,4 +1,4 @@
-﻿using Microsoft.VisualStudio.Services.Agent.Util;
+using Microsoft.VisualStudio.Services.Agent.Util;
```

Comment:
undo the changes.

Reply:
This is already undone. 
Was removed by mistake in previous commit, and then re-added it back.